### ISSUE TITLE
In the Multilingual word Embeddings, the Arabic file is not correct

### ISSUE THREAD

**Issue by mamprogr** [facebookresearch/MUSE#150]
In the Multilingual word Embeddings section, the Arabic file is not correct. It is only ~417MB and there is no \n at the end of the file like other languages.

**Comment by adodge** [comment]
This is true, and still true.  The file has been truncated.  The last line has fewer than 300 dimensions.

**Comment by dnozza** [comment]
The file is still corrupted.
